include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -0,0 +1,347 @@
+                                        {
+                                            throw parse_error{std::string{"Encountered an unexpected letter: "} +
+                                                              is_legal_alph.msg.string() +
+                                                              " evaluated to false on " +
+                                                              detail::make_printable(c)};
+                                        }
+                                        return c;
+                                    })
+                                  | view::char_to<value_type_t<seq_type>>,         // convert to actual target alphabet
+                         std::back_inserter(sequence));
+        }
+        else
+        {
+            detail::consume(stream_view | view::take_until(is_end));
+        }
+        //Jump over //
+        detail::consume(stream_view | view::take_until(is_cntrl));
+        ++stream_it;
+
+
+        // make sure "buffer at end" implies "stream at end"
+        if ((std::istreambuf_iterator<char>{stream} == std::istreambuf_iterator<char>{}) &&
+            (!stream.eof()))
+        {
+            stream.get(); // triggers error in stream and sets eof
+        }
+    }
+
+    //!\copydoc sequence_file_output_format_concept::write
+    template <typename stream_type,     // constraints checked by file
+              typename seq_type,        // other constraints checked inside function
+              typename id_type,
+              typename qual_type>
+    void write(stream_type                     & stream,
+               sequence_file_output_options const & options,
+               seq_type                       && sequence,
+               id_type                        && id,
+               qual_type                      && SEQAN3_DOXYGEN_ONLY(qualities))
+    {
+
+        ranges::ostreambuf_iterator stream_it{stream};
+        size_t sequence_size = 0;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+            sequence_size = ranges::size(sequence);
+
+        // ID
+        if constexpr (detail::decays_to_ignore_v<id_type>)
+        {
+            throw std::logic_error{"The ID field may not be set to ignore when writing embl files."};
+        }
+        else
+        {
+            if (options.complete_header)
+            {
+                ranges::copy(id, stream_it);
+            }
+            else
+            {
+                if (ranges::empty(id)) //[[unlikely]]
+                    throw std::runtime_error{"The ID field may not be empty when writing embl files."};
+
+                stream_it = 'I';
+                stream_it = 'D';
+                stream_it = ' ';
+                ranges::copy(id, stream_it);
+                stream_it = ';';
+                stream_it = ' ';
+                ranges::copy(std::to_string(sequence_size), stream_it);
+                ranges::copy(std::string_view{" BP.\nXX\n"}, stream_it);
```

Comment:
Question: Is the line consisting of two 'X' part of the format specification? 
You introduce **always** multi-line ids with this. And when you parse, you do not enforce them.

Reply:
The XX are optional, I don't remember why I put them there, I will delete this.